charts/seed-controlplane/charts/cluster-autoscaler/templates/deployment.yaml

```diff
@@ -0,0 +1,65 @@
+---
+apiVersion: {{ include "deploymentversion" . }}
+kind: Deployment
+metadata:
+  name: cluster-autoscaler
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: kubernetes
+    role: cluster-autoscaler
+spec:
+  revisionHistoryLimit: 0
+  replicas: {{ .Values.replicas }}
+  selector:
+    matchLabels:
+      app: kubernetes
+      role: cluster-autoscaler
+  template:
+    metadata:
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+{{- if .Values.podAnnotations }}
+{{ toYaml .Values.podAnnotations | indent 8 }}
+{{- end }}
+      labels:
+        app: kubernetes
+        role: cluster-autoscaler
+    spec:
+      serviceAccountName: cluster-autoscaler
+      terminationGracePeriodSeconds: 5
+      containers:
+      - name: cluster-autoscaler
+        image: {{ index .Values.images "cluster-autoscaler" }}
+        imagePullPolicy: IfNotPresent
+        command:
+        - ./cluster-autoscaler
+        - --kubeconfig=/var/lib/cluster-autoscaler/kubeconfig
+        - --cloud-provider=mcm
+        {{- range $key, $pool := .Values.workerPools }}
+        - --nodes={{ $pool.min }}:{{ $pool.max }}:{{ $.Release.Namespace }}.{{ $pool.name }}
+        {{- end }}
+        - --stderrthreshold=info
+        - --skip-nodes-with-system-pods=false
+        - --skip-nodes-with-local-storage=false
+        - --expander=least-waste
+        - --v=5
```

Comment:
It would be better to keep this log level to --v=2 ? With v==5, there are too many logs and we might loose older error logs if we keep them.